packages/adp-tooling/src/translations/adp-tooling.i18n.json

```diff
@@ -52,27 +52,27 @@
-        "errorDuplicatedValueSapui5Model": "SAPUI5 Model with the same name was already added to the project",
-        "errorDuplicateValueComponentId": "Component usage with the same name was already added to the project",
-        "errorDuplicateValueLibrary": "Library with the same name was already added to the project",
-        "errorDuplicateNamesOData": "OData Service Name must be different from OData Annotation Data Source Name",
-        "errorInputInvalidValuePrefix": "{{value}} should start with '{{prefix}}'",
-        "errorCustomerEmptyValue": "{{value}} should contain at least one character in addition to '{{prefix}}'",
+        "missingIconOrTitleOrSubtitle": "At least one of the Title, Subtitle, or Icon fields must have a value.",
+        "fileDoesNotExist": "This file does not exist.",
+        "annotationFileAlreadyExists": "There is already an annotation file with the same name. Please choose another file or rename the file and select it again.",
+        "errorDuplicatedValueOData": "An OData annotation or service with the same name was already added to the project. Rename and try again.",
+        "errorDuplicatedValueSapui5Model": "An SAPUI5 model with the same name was already added to the project. Rename and try again.",
+        "errorDuplicateValueComponentId": "A component usage with the same name was already added to the project. Rename and try again.",
+        "errorDuplicateValueLibrary": "A library with the same name was already added to the project. Rename and try again.",
+        "errorDuplicateNamesOData": "An OData Service Name must be different from an OData Annotation Data Source Name. Rename and try again.",
+        "errorInputInvalidValuePrefix": "{{value}} must start with '{{prefix}}'.",
+        "errorCustomerEmptyValue": "{{value}} must contain at least one character in addition to '{{prefix}}'.",
         "errorInvalidDataSourceURI": "Invalid URI. The URI must start and end with '/' and contain no spaces.",
-        "appDoesNotSupportManifest": "The selected application is not supported by SAPUI5 Adaptation Project as it does not have a manifest.json file. Please select a different application.",
-        "ui5VersionNotReachableError": "The URL of the SAPUI5 version you have selected is not reachable. The URL <URL> should be made accessible through cloud connector and the destination configuration so it can be consumed within the SAPUI5 adaptation project and its SAPUI5 Adaptation Editor.",
+        "appDoesNotSupportManifest": "The selected application is not supported by SAPUI5 adaptation projects as it does not have a `manifest.json` file. Please select a different application.",
+        "ui5VersionNotReachableError": "The URL of the SAPUI5 version you have selected is not reachable. The <URL> URL must be made accessible through cloud connector and the destination configuration so it can be consumed within the SAPUI5 adaptation project and its SAPUI5 Adaptation Editor.",
```

Comment:
```suggestion
        "ui5VersionNotReachableError": "The URL of the SAPUI5 version you have selected is not reachable. The <URL> URL must be made accessible through the cloud connector and the destination configuration so it can be consumed within the SAPUI5 adaptation project and its SAPUI5 Adaptation Editor.",
```